### PR TITLE
Fix bugs

### PR DESCRIPTION
**Pull request by weradsaoud**

### CODE REVIEW

File: src/modules/administration/popups/violation-penalty-popup/violation-penalty-popup.component.ts [hunk 1]
```diff
@@ -248,10 +252,28 @@ export class ViolationPenaltyPopupComponent
         })
         .subscribe(data => {
           this.filteredPenalties = data;
+          this.penaltyGuidanceDisplay();
```
Comment (aalshahin) [inline src/modules/administration/popups/violation-penalty-popup/violation-penalty-popup.component.ts:255]: rename the function to filterPenaltyGuidances

File: src/validators/validation-utils.ts [hunk 2]
```diff
@@ -187,7 +187,36 @@ export function minlengthValidator(minLength: number): ValidatorFn {
       : null;
   };
 }
+export function maxValueValidator(maxValue: number): ValidatorFn {
+  if (!isValidValue(maxValue) || maxValue <= 0) {
```
Comment (aalshahin) [inline src/validators/validation-utils.ts:191]: remove maxValue <=0
maybe sometime we need max value to be less than 0

File: src/validators/validation-utils.ts [hunk 3]
```diff
@@ -187,7 +187,36 @@ export function minlengthValidator(minLength: number): ValidatorFn {
...
+      return null;
+    }
+    const isNumber = /^[0-9\u0660-\u0669]+$/.test(control.value);
+    if (!isNumber) return { number: true };
+    const numberValue = Number(control.value);
+    return numberValue <= maxValue ? null : { max: { max: maxValue } };
+  };
+}
+export function minValueValidator(minValue: number): ValidatorFn {
+  if (!isValidValue(minValue) || minValue <= 0) {
```
Comment (aalshahin) [inline src/validators/validation-utils.ts:206]: remove minValue <= 0
for same reason

**Review by aalshahin — CHANGES_REQUESTED** [review]
make sure that everything is working fine after your changes

File: src/modules/administration/popups/violation-penalty-popup/violation-penalty-popup.component.ts (hunk 1 above)
Comment (weradsaoud) [reply src/modules/administration/popups/violation-penalty-popup/violation-penalty-popup.component.ts:255]: done

File: src/validators/validation-utils.ts (hunk 2 above)
Comment (weradsaoud) [reply src/validators/validation-utils.ts:191]: done

File: src/validators/validation-utils.ts (hunk 3 above)
Comment (weradsaoud) [reply src/validators/validation-utils.ts:206]: done

**Pull request closed without merging**